Automattic/simplenote-electron
# Dark Theme Tweaks

I'd like to propose we change the dark them to match up with the recent changes we've made to the iOS and macOS apps. It essentially desaturates the 'blueness' of the dark theme, and increases the contrast by making the text a bit lighter and the background color darker. The border color is also way more subtle than before.

Before:
<img width="969" alt="screen shot 2018-11-30 at 3 05 53 pm" src="https://user-images.githubusercontent.com/789137/49319434-b487fd80-f4b1-11e8-87c1-b8601bad9440.png">

After:
<img width="987" alt="screen shot 2018-11-30 at 3 09 27 pm" src="https://user-images.githubusercontent.com/789137/49319492-03359780-f4b2-11e8-9758-12f9e656c800.png">

**To Test**
* Enable the dark theme, and explore the app to ensure everything looks good!
* Also ensure that the light theme still appears correctly (It uses the `$gray*` variables which were slightly tweaked in this PR)
